code/service/src/main/resources/application.conf

```diff
@@ -40,12 +40,20 @@ akka {
 		provider = cluster
 		serializers {
 			proto = "akka.remote.serialization.ProtobufSerializer"
-			cmdSerializer = "com.namely.chiefofstate.CommandSerializer"
+			jackson-cbor = "akka.serialization.jackson.JacksonCborSerializer"
+			cmdSerde = "com.namely.chiefofstate.CommandSerializer"
+			scalapb = "com.namely.chiefofstate.ScalapbSerializer"
 		}
 		serialization-bindings {
-			# state is serialized using protobuf
-			"scalapb.GeneratedMessage" = proto
-			"com.namely.chiefofstate.AggregateCommand" = cmdSerializer
+			"java.io.serializable" = none
+			"scalapb.GeneratedMessage" = scalapb
+			"com.namely.chiefofstate.AggregateCommand" = cmdSerde
+			"com.google.protobuf.Message" = proto
+			"com.namely.chiefofstate.CborSerializable" = jackson-cbor
+		}
+
+		serialization-identifiers {
+			"com.namely.chiefofstate.ScalapbSerializer" = 10000
```

Comment:
since we are using a custom serde anyways, lets just make it use proto serialization.